### ISSUE TITLE
Publish latest version

### ISSUE THREAD

**Issue by Vincz** [frontend-collective/react-image-lightbox#123]
Hi guys!
Would you mind to publish the latest update on npm?
I'm using your lib with nextjs and need the latest umd fix in order to make it work :)
Thanks !

**Comment by rdfriedl** [comment]
Im using react-static and Iv also run into the same issue.

It looks like UMD part of [dist/main.js](https://unpkg.com/react-image-lightbox@5.0.0/dist/main.js) in the latest 5.0.0 is still using webpacks old UMD
I tried cloning the repo locally and running `npm build` 

current 5.0.0:
```js
!function(root, factory) {
    "object" == typeof exports && "object" == typeof module ? module.exports = factory() : "function" == typeof define && define.amd ? define([], factory) : "object" == typeof exports ? exports.ReactImageLightbox = factory() : root.ReactImageLightbox = factory();
}(window, function() {
```
local build ( with all packages matching package-lock.json `npm@6.2.0` )
```js
!function(root, factory) {
    "object" == typeof exports && "object" == typeof module ? module.exports = factory() : "function" == typeof define && define.amd ? define([], factory) : "object" == typeof exports ? exports.ReactImageLightbox = factory() : root.ReactImageLightbox = factory();
}("undefined" != typeof self ? self : this, function() {
```

the main difference is the reference to `self` instead of `window`.

**Comment by wuweiweiwu** [comment]
Will do. #126 is going to fix this

**Comment by sugarshin** [comment]
Any updates? Not includes latest files for now. (e.g., `index.d.ts` )

**Comment by rdfriedl** [comment]
Nothing yet, although @Resumane forked this repo and has a version with the fix in it :smiley: 
`npm install --save Resumane/react-image-lightbox#fix-umd-bug`
https://github.com/frontend-collective/react-image-lightbox/issues/152#issuecomment-438526122

**Comment by wuweiweiwu** [comment]
Ill take some time this week to push out #126 and then release :)

**Comment by wuweiweiwu** [comment]
published in `5.1.0`!!!!!
